### DISCUSSION TITLE
Google Drive Syncing

### DISCUSSION THREAD

**Discussion by vedxp** [anyproto/contributors#48] [Anytype apps]
Hi Anytype team,

I am a big fan of Anytype and I find it to be very useful for note-taking and project management.

However, I have run into a problem. I have used up all of my 1 GB of free storage space on the Anytype servers. This means that my notes are now only stored locally on my Android smartphone.

I would like to request a feature that would allow me to backup my notes to my Google Drive account. This would be a great way to ensure that my data is not lost.

I know that Anytype is a privacy-focused app and you may be concerned about storing user data on Google Drive. However, I believe that this feature would be a valuable addition to the app and it would not compromise the privacy of users.

Google Drive is a secure platform and users can choose to encrypt their data before uploading it to Drive. This would ensure that the data is only accessible to the user.

I hope you will consider adding this feature to Anytype. I believe that it would be a great way to improve the app and it would make it more user-friendly.

Thank you for your time and consideration.

**Comment by ahmedwael216** [comment]
As I understand it, the 1GB limit is for files, not native AnyType objects.

So If your notes are not PDF files and are just normal object they *should* sync.

**Reply by vedxp** [reply]
I see, thanks for the clarification :)

**Comment by Seanytype** [comment]
@ahmedwael216 is correct, the storage limit has no impact on objects (∴notes), only external files imported into Anytype. Regardless of back up storage consumption, you can keep an unlimited amount of data locally on your device, and files should sync peer-to-peer (p2p) as long as your devices are on the same network. 

 ~~In any case, you can apply for a small storage increase by writing to ---- and filling out the survey.~~
 **Update:** the storage reward program is no longer in effect.

**Reply by vedxp** [reply]
Got it, thanks for the information :)

I have one more question: is there a way to export all of my notes/data from Anytype? I was unable to find the directory in my file system where Anytype saves the notes.

For reference, Obsidian saves all of its data in the directory that I choose in my file system.

**Reply by Biepa** [reply]
@vedxp The thing here is that Anytype is not saving your data in an easily readable format.
You can currently export everything by clicking on your space and choose export.
A discussion about backing up data is currently open here: https://community.anytype.io/t/how-do-you-backup-your-data/10280
And here is a feature request to make this possible automatically: https://community.anytype.io/t/auto-backup-auto-export/9666

**Reply by vedxp** [reply]
I understand. However, I couldn't find the export option in the Android version of Anytype. Is there a way to export my notes on Android?

Also, I couldn't find the directory where Anytype saves the notes in my file system. In Obsidian, users can choose the directory where the notes are saved. If Anytype had the same feature, users could set up auto-sync for that directory to any cloud storage, avoiding manual exporting and uploading.

**Reply by vedxp** [reply]
@Biepa I am wondering if you have any information about this.

**Reply by Biepa** [reply]
@vedxp I waited and hoped someone from the official team would respond on this :sweat_smile: (I'm only a community mod).

While not being adapted to the latest versions, here is feature parity list of the platforms.
https://doc.anytype.io/d/anytype-basics/feature-list-by-platform

Regarding the export option on Android. There are quite a few features that are not available on the mobile apps but are on Desktop.
With a growing amount of objects I also think that personally I would never export all my data on a mobile device. This may also be due to the fact, that currently mobile devices are only loading what you open. They are not having a full copy of all your data by default.
Though this is a feature request multiple times already, so I don't see why an export shouldn't be possible then (not speaking for the devs as I don't have any clue about the backend of all of that).

I don't know where the data is stored directory wise and also can't comment if a setup with a 3rd party sync solution would work like you expect it from a single file based setup like obsidian. That would need an official comment and maybe some testing.

**Reply by vedxp** [reply]
I appreciate the information about the feature parity list and the fact that an export option is a feature request.

I understand that the mobile apps don't have all of the features of the desktop app, and I agree that it's not ideal to export all of my data on a mobile device. However, I would like to have the option to export my notes if I need to, especially if I'm switching devices or if I want to back up my data.

I'll keep an eye out for any updates on these features. In the meantime, thanks again for your help.

> I don't know where the data is stored directory wise and also can't comment if a setup with a 3rd party sync solution would work like you expect it from a single file based setup like obsidian. That would need an official comment and maybe some testing.

I'll be waiting to hear back from the developers on this.

**Reply by vedxp** [reply]
> @ahmedwael216 is correct, the storage limit has no impact on objects (∴notes), only external files imported into Anytype. Regardless of back up storage consumption, you can keep an unlimited amount of data locally on your device, and files should sync peer-to-peer (p2p) as long as your devices are on the same network.
> 
> In any case, you can apply for a small storage increase by writing to [storage@anytype.io](mailto:storage@anytype.io) and filling out the survey.

Seems I missed the opportunity to increase my storage space.
I got this reply for my request to increase the storage increase: 
```html
Hi there!

Thank you for reaching out.

Please note that applying for storage increases through this inbox is no longer available. We're thrilled to introduce several membership options tailored to provide increased storage, multiplayer capabilities, and more. You can find all the details in our Pricing Guide.

Cheers!

Anyteam

(The Anyteam works Mon-Fri, 10 a.m.- 6 p.m. CET. Please allow sufficient time to review your request.)

If you're eligible for Alpha or Nightly Storage rewards, simply provide your Technical info (Help>Technical Info), and your Community forum username.
```

#### Could you please advise me on how to increase my storage space at this point, for free? 
#### That'll mean a lot!

**Reply by vedxp** [reply]
@Seanytype any update on the above request?

**Reply by Seanytype** [reply]
@vedxp As mentioned in our email reply, the storage reward program was temporary and is no longer available. Right now, the only way to increase your storage is by subscribing to a paid tier. If we launch a similar reward program in the future, we’ll make an official announcement. Thanks!

**Comment by vedxp** [comment]
> And here is a feature request to make this possible automatically: https://community.anytype.io/t/auto-backup-auto-export/9666

I can't seem to view this thread. It might be private.

I also wasn't able to create an account on the Anytype community. There's no signup page, and I couldn't find any information about it in the Android version of Anytype.

**Reply by Biepa** [reply]
Sign up with this link: https://community.anytype.io/invites/GJHiWdNuXk

@Seanytype Couldn't find an invite link option in iOS either.

**Reply by vedxp** [reply]
That you so much :)

**Comment by darshansharma** [comment]
Why on Google Cloud? 
We can provide options whether to store on S3, Google Cloud Bucket or Azure 
OR
we can by default use S3 for additional storage as it is best in the industry. 
BackBlaze - Another option - It's cheaper but trustworthiness score is low.
